backend/api/pretix/types.py

```diff
@@ -165,6 +167,12 @@ def type(self) -> TicketType:
 
         return TicketType.STANDARD
 
+    @strawberry.field
+    def sold_out(self) -> Optional[bool]:
```

Comment:
I removed the custom `Ticket` type in the frontend to use those generated by the backend, so soldOut was calculated on the frontend. I left the calculation in the frontend but I added this field so typescript will not complain and later we can use this instead...

Reply:
What problem did you have?

Reply:
The build was failing in the typescript checks because I was passing `TicketItem` down to `QuestionSection`, we didn't have all these types in the backend before I just created them... so Typescript was complaining `Ticket` is not compatible with `TicketItem` all over... 
or stuff like `soldOut` is not in `TicketItem`.... etc etc